Contributing.md

```diff
@@ -0,0 +1,25 @@
+# Contribute to Nylas
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+The following is a set of guidelines for contributing to the Nylas Java SDK; these are guidelines, not rules, so please use your best judgement and feel free to propose changes to this document via pull request.
+
+# How to Ask a Question
+
+If you have a question about how to use the Java SDK, please [create an issue](https://github.com/nylas/nylas-java/issues) and label it as a question. If you have more general questions about the Nylas Communications Platform, or the Nylas Email, Calendar, and Contacts API, please reach out to support@nylas.com to get help.
+
+# How To Contribute
+## Report a Bug or Request a Feature
+
+If you encounter any bugs while using this software, or want to request a new feature or enhancement, please [create an issue](https://github.com/nylas/nylas-java/issues) to report it, and make sure you add a label to indicate what type of issue it is.
+
+## Contribute Code
+
+Pull requests are welcome for bug fixes. If you want to implement something new, [please request a feature](https://github.com/nylas/nylas-java/issues) first so we can discuss it.
+
+While writing your code contribution, make sure you follow the testing conventions found in the [tests directory](https://github.com/nylas/nylas-java/tree/master/tests) for any components that you add. We use [codecov](https://codecov.io/gh/nylas/nylas-java) to test coverage, please ensure that your contributions don’t cause a decrease to test coverage.
```

Comment:
We are still lacking in automated tests and I have not hooked up any code coverage analysis tools.

Reply:
I've removed this content from the file.